src/domUpdates.js

```diff
@@ -35,16 +40,60 @@ let domUpdates = {
       <section class='booked-info-card'>
         <p>Booking For ${booking.date}</p>
         <p>Confirmation ID: ${booking.id}</p>
-        <p>Room Type ${foundRoom.roomType}</p>
-        <p>Cost ${foundRoom.costPerNight}</p>
+        <p>Room Type: ${foundRoom.roomType}</p>
+        <p>Cost: $ ${foundRoom.costPerNight}</p>
       </section>
       `
     })
+  },
+
+  displayAvailableRooms(currentCustomer, roomsData, bookingsData) {
+    availableRoomsSection.innerHTML = '';
+    const date = selectedDate.value.split('-').join('/');
+    currentCustomer.getAvailableRooms(date, roomsData, bookingsData);
+    console.log('customer', currentCustomer)
+
+    console.log('customer', currentCustomer.availableRooms)
+    currentCustomer.availableRooms.forEach((room) => {
+      availableRoomsSection.innerHTML += `
+      <section class='individual-room-cards'>
```

Comment:
I would add individual-room-cards to the one you used for booked-info-card, so that the same styling applies to your available rooms.  Right now individual rooms all run together and is hard to read.

Reply:
YES! That is 100% the plan. It's impossible to read now and just ew. I randomly threw the styling on the one and kinda like it, but I didn't think to throw the same class onto that styling before I pushed it. Dumb of me, but 100%, for sure!